### PR TITLE
Fix #1 Added a responsive home page to the website with some designs 

### LINKED ISSUE #1: Homepage of the website
Create a homepage of the website in reference to the original website: www.technonjr.org.
Things to keep in mind:
1. The page should be responsive that is it should work well on both mobiles and desktop screens. 
2. Good coding practice should be followed.
3. The Pull request need not be in a single commit.

### PR DESCRIPTION
**Pull request by m-ohit-s**
## Explanation
This PR solves the #1 issue.
Added a home page to the existing index.html
Please check and suggest if any changes required.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR is made from a branch that's **not** called "master".
- [x] The PR is made from a branch that is up-to-date with "master".
- [x] The PR's branch is based on "master" and not on any other branch.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+<head>
+  <!-- Required meta tags -->
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!-- Bootstrap CSS -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+  <title>HOME - Techno India NJR Institute Of Technology</title>
+</head>
```
Comment (jkugsiya) [inline index.html:13]: Carousel is not working because jquery and bootstrap js is not linked with the html

File: index.html [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+        <img class="d-block w-100" src="http://technonjr.org/wp-content/uploads/2015/03/akshay-copy1.jpg" alt="Akshay Nandwana at Google IO">
+      </div>
+      <div class="carousel-item">
+        <img class="d-block w-100" src="http://technonjr.org/wp-content/uploads/2015/03/slider00011.jpg " alt="Techno India NJR Udaipur Campus">
+      </div>
+    </div>
+  </div>
+
+  <!--achievements-->
+<section id="achievements" style="padding-top:10%;">
```
Comment (jkugsiya) [inline index.html:40]:
```suggestion
<section id="achievements">
```
use external css instead

File: index.html [hunk 3]
```diff
@@ -0,0 +1,462 @@
...
+  <div class="container">
+    <div class="row">
+      <div class="col-md-7">
+        <div class="card mb-3" style="min-height:20px; box-shadow:10px 5px 10px #888888">
```
Comment (jkugsiya) [inline index.html:44]:
```suggestion
        <div class="card mb-3">
```
same- use external css

File: index.html [hunk 4]
```diff
@@ -0,0 +1,462 @@
...
+            <div class="card-text">
+              <h2><b><i>NEWS</i></b></h2>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-7">
+        <div class="card mb-3" style="box-shadow:none;">
```
Comment (jkugsiya) [inline index.html:55]:
```suggestion
        <div class="card mb-3">
```
same

File: index.html [hunk 5]
```diff
@@ -0,0 +1,462 @@
...
+          <div class="card-body">
+            <div class="card-text">
+              <article>
+                <h4><a href="http://technonjr.org/news/techno-india-njr-is-participating-in-the-nvidia-gtc-conference-on-april-12th-2021/" title="Techno India NJR is participating in the NVIDIA GTC conference on April 12th, 2021.">Techno India NJR is participating in the NVIDIA GTC conference on April 12th, 2021.</a></h4>
```
Comment (jkugsiya) [inline index.html:59]:
```suggestion
                <h4><a href="" title="Techno India NJR is participating in the NVIDIA GTC conference on April 12th, 2021.">Techno India NJR is participating in the NVIDIA GTC conference on April 12th, 2021.</a></h4>
```
Remove this link, we have to create our own instead

File: index.html [hunk 6]
```diff
@@ -0,0 +1,462 @@
...
+                <small>NVIDIA GTC is a global virtual conference and will...</small><hr>
+              </article>
+              <article>
+                <h4><a href="http://technonjr.org/news/placement-2020-21-cognizant-technologies-selects-3-techno-njr-students-at-a-package-of-5-41-lpa/" title="[PLACEMENT 2020-21] Cognizant Technologies Selects 3 Techno NJR students at a Package of 5.41 LPA.">[PLACEMENT 2020-21] Cognizant Technologies Selects 3 Techno NJR students at a Package of 5.41 LPA.</a></h4>
```
Comment (jkugsiya) [inline index.html:63]:
```suggestion
                <h4><a href="" title="[PLACEMENT 2020-21] Cognizant Technologies Selects 3 Techno NJR students at a Package of 5.41 LPA.">[PLACEMENT 2020-21] Cognizant Technologies Selects 3 Techno NJR students at a Package of 5.41 LPA.</a></h4>
```
Same

File: index.html [hunk 7]
```diff
@@ -0,0 +1,462 @@
...
+                <small>[PLACEMENT 2020-21] Cognizant Technologies Selects...</small><hr>
+              </article>
+              <article>
+                <h4><a href="http://technonjr.org/news/thermax-selects-4-techno-njr-students/" title="Thermax selects 4 Techno NJR Students">Thermax selects 4 Techno NJR Students</a></h4>
```
Comment (jkugsiya) [inline index.html:67]:
```suggestion
                <h4><a href="">Thermax selects 4 Techno NJR Students</a></h4>
```
Same here

File: index.html [hunk 8]
```diff
@@ -0,0 +1,462 @@
...
+                <small>Congratulations to the selected students: 1. Chetan...</small><hr>
+              </article>
+              <article>
+                <h4><a href="http://technonjr.org/news/startup-venture-wricks-by-ex-students-of-techno-india-njr-incubated-at-techno-india-njr-incubation-center-has-won-the-second-prize-of-canadian-12500/" title="Startup Venture Wricks by ex-students of Techno India NJR incubated at Techno India NJR Incubation Center has won the second prize of Canadian $12500/-">Startup Venture Wricks by ex-students of Techno India NJR incubated at Techno India NJR Incubation Center has won the second prize of Canadian $12500/-</a></h4>
```
Comment (jkugsiya) [inline index.html:71]:
```suggestion
                <h4><a href="" title="Startup Venture Wricks by ex-students of Techno India NJR incubated at Techno India NJR Incubation Center has won the second prize of Canadian $12500/-">Startup Venture Wricks by ex-students of Techno India NJR incubated at Techno India NJR Incubation Center has won the second prize of Canadian $12500/-</a></h4>
```
Same

File: index.html [hunk 9]
```diff
@@ -0,0 +1,462 @@
...
+              </article>
+            </div>
+          </div>
+        </div>
+      </div>
+  </div>
+  <div class="row">
+    <div class="col-md-5">
+      <a href="http://technonjr.org/news" class="btn btn-lg btn-secondary">READ MORE</a>
+    </div>
```
Comment (jkugsiya) [inline index.html:82]:
```suggestion
      <a href="" class="btn btn-lg btn-secondary">READ MORE</a>
    </div>
```
Don't use this links of external website. We have to create our own pages later instead.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,462 @@
...
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-5">
+      </div>
+      <div class="col-md-7">
+        <div class="card text-center mb-3" style="box-shadow:none; min-height:20px;">
+            <img src="http://technonjr.org/wp-content/uploads/2015/03/website-poster-1024x576.jpg" class="card-image-top" alt="International Research">
+        </div>
```
Comment (jkugsiya) [inline index.html:105]:
```suggestion
            <img src="" class="card-image-top" alt="International Research">
        </div>
```
Download and keep the image in images folder instead using them from external website

File: index.html [hunk 11]
```diff
@@ -0,0 +1,462 @@
...
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-5">
+      </div>
+      <div class="col-md-7">
+        <a href="http://technonjr.org/department/departments/research/" class="btn btn-lg btn-secondary">READ MORE</a>
+      </div>
```
Comment (jkugsiya) [inline index.html:113]:
```suggestion
        <a href="">READ MORE</a>
      </div>
```
Remove link to external website

File: index.html [hunk 12]
```diff
@@ -0,0 +1,462 @@
...
+  <div class="container">
+    <div class="title">
+      <h1>DEPARTMENTS</h1>
+      <hr>
+    </div>
+    <div class="row">
+      <div class="col-md-4">
+        <div class="card text-center mb-3">
+          <img class="details card-image-top" src="http://technonjr.org/wp-content/uploads/2014/10/computer-science-550x400.jpg" alt="Computer Science Engineering &#038; Information Technology (CSE &#038; IT)">
+          <div class="image__overlay image__overlay--blur">
```
Comment (jkugsiya) [inline index.html:130]:
```suggestion
          <img class="details card-image-top" src="" alt="Computer Science Engineering &#038; Information Technology (CSE &#038; IT)">
          <div class="image__overlay image__overlay--blur">
```
Download and use local image here

File: index.html [hunk 13]
```diff
@@ -0,0 +1,462 @@
...
+            <p class="image__description">
+              <a title="Computer Science Engineering &#038; Information Technology (CSE &#038; IT)" href="http://technonjr.org/department/cse/ ">
+                Computer Science Engineering &#038; Information Technology (CSE &#038; IT)
```
Comment (jkugsiya) [inline index.html:133]:
```suggestion
              <a title="Computer Science Engineering &#038; Information Technology (CSE &#038; IT)" href="">
                Computer Science Engineering &#038; Information Technology (CSE &#038; IT)
```
Remove links to the external website

File: index.html [hunk 14]
```diff
@@ -0,0 +1,462 @@
...
+              </a>
+            </p>
+          </div>
+          <div class="card-body">
+            <h5 class="card-text">
+              <a title="Computer Science Engineering &#038; Information Technology (CSE &#038; IT)" href="http://technonjr.org/department/cse/ ">
+                Computer Science Engineering &#038; Information Technology (CSE &#038; IT)
```
Comment (jkugsiya) [inline index.html:140]:
```suggestion
              <a title="Computer Science Engineering &#038; Information Technology (CSE &#038; IT)" href="">
                Computer Science Engineering &#038; Information Technology (CSE &#038; IT)
```
Same

File: index.html [hunk 15]
```diff
@@ -0,0 +1,462 @@
...
+              </a>
+            </h5>
+          </div>
+        </div>
+      </div>
+      <div class="col-md-4">
+        <div class="card text-center mb-3">
+          <img src="http://technonjr.org/wp-content/uploads/2014/10/electricals-550x400.jpg" class="details card-image-top" alt="Electronics and Communication Engineering (ECE)">
+          <div class="image__overlay image__overlay--blur">
```
Comment (jkugsiya) [inline index.html:149]:
```suggestion
          <img src="" class="details card-image-top" alt="Electronics and Communication Engineering (ECE)">
          <div class="image__overlay image__overlay--blur">
```
Download the image

File: index.html [hunk 16]
```diff
@@ -0,0 +1,462 @@
...
+            <p class="image__description">
+              <a title="Electronics and Communication Engineering (ECE)" href="http://technonjr.org/department/electronics-and-communication-engineering-ece/ ">
```
Comment (jkugsiya) [inline index.html:151]:
```suggestion
              <a title="Electronics and Communication Engineering (ECE)" href=" ">
```
Download and use images

File: index.html [hunk 17]
```diff
@@ -0,0 +1,462 @@
...
+                Electronics and Communication Engineering (ECE)
+              </a>
+            </p>
+          </div>
+          <div class="card-body">
+            <h5 class="card-text">
+              <a title="Electronics and Communication Engineering (ECE)" href="http://technonjr.org/department/electronics-and-communication-engineering-ece/ ">
```
Comment (jkugsiya) [inline index.html:158]:
```suggestion
              <a title="Electronics and Communication Engineering (ECE)" href="">
```
Remove link to external website

File: index.html [hunk 18]
```diff
@@ -0,0 +1,462 @@
...
+                Electronics and Communication Engineering (ECE)
+              </a>
+            </h5>
+          </div>
+        </div>
+      </div>
+      <div class="col-md-4">
+        <div class="card text-center mb-3">
+          <img src="http://technonjr.org/wp-content/uploads/2014/09/mechanical-550x400.jpg" class="details card-image-top" alt="Mechanical Engineering (ME)">
+          <div class="image__overlay image__overlay--blur">
```
Comment (jkugsiya) [inline index.html:168]:
```suggestion
          <img src="">
          <div class="image__overlay image__overlay--blur">
```
Download image

File: index.html [hunk 19]
```diff
@@ -0,0 +1,462 @@
...
+            <p class="image__description">
+              <a title="Mechanical Engineering (ME)" href="http://technonjr.org/department/mechanical-engineering-me/ ">
+                Mechanical Engineering (ME)
```
Comment (jkugsiya) [inline index.html:171]:
```suggestion
              <a title="Mechanical Engineering (ME)" href=" ">
                Mechanical Engineering (ME)
```
Don't use link to external website

File: index.html [hunk 20]
```diff
@@ -0,0 +1,462 @@
...
+              </a>
+            </p>
+          </div>
+          <div class="card-body">
+            <h5 class="card-text">
+              <a title="Mechanical Engineering (ME)" href="http://technonjr.org/department/mechanical-engineering-me/ ">
+                Mechanical Engineering (ME)
```
Comment (jkugsiya) [inline index.html:178]:
```suggestion
              <a title="Mechanical Engineering (ME)" href="">
                Mechanical Engineering (ME)
```
Same

File: index.html [hunk 21]
```diff
@@ -0,0 +1,462 @@
...
+              </a>
+            </h5>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-4">
+        <div class="card text-center mb-3">
+          <img src="http://technonjr.org/wp-content/uploads/2014/10/electronics-550x400.jpg" class="details card-image-top" alt="Electrical Engineering (EE)">
```
Comment (jkugsiya) [inline index.html:188]:
```suggestion
          <img src="">
```
Same

**Review by jkugsiya — CHANGES_REQUESTED** [review]
Left some changes to be done. Also look for all the same problem occurrences in the code and fix them.

File: index.html (hunk 8 above)
Comment (m-ohit-s) [reply index.html:71]: Should i leave the links empty for now

**Comment by m-ohit-s** [comment]
Should I Leave the links empty for now

**Comment by jkugsiya** [comment]
> Should I Leave the links empty for now

Yes you can leave the links empty

File: index.html [hunk 22]
```diff
@@ -0,0 +1,685 @@
...
+              <div class="col-md-4">
+                <a class="btn btn-secondary btn-lg ds_buttons" href="#">Download E-Brochure 2015</a>
+              </div>
+              <div class="col-md-4">
+                <a class="btn btn-secondary btn-lg ds_buttons" href="#">Admission Online</a>
+              </div>
+            </div>
+          </div>
+        </section>
+      </div>
```
Comment (jkugsiya) [inline index.html:670]: These buttons and heading don't look good on small screen sizes:
image

**Review by jkugsiya — CHANGES_REQUESTED** [review]
PTAL @m-ohit-s. Left some comments and changes to be done.

**Comment by jkugsiya** [comment]
Your PR needs to update with the upstream master and also include the footer now as it is available.

**Review by jkugsiya — APPROVED** [review]
LGMT, @m-ohit-s! Approving changes

**Pull request merged**